--- a/target/linux/qualcommax/image/ipq60xx.mk
+++ b/target/linux/qualcommax/image/ipq60xx.mk
@@ -151,6 +151,13 @@ define Device/link_nn6000-v2
 endef
 TARGET_DEVICES += link_nn6000-v2
 
+define Device/link_nn6000-v2-12m
	$(Device/link_nn6000-common)
	DEVICE_MODEL := NN6000 v2
	DEVICE_VARIANT := v2
endef
TARGET_DEVICES += link_nn6000-v2-12m
+
 define Device/linksys_mr
 	$(call Device/FitImage)
 	DEVICE_VENDOR := Linksys
